docs/docs/getting-started/6-nocode-docker-deployment.md

```diff
@@ -0,0 +1,38 @@
+# No-Code Deployment with Docker
+
+This is the easiest way to deploy your Groundlight detector on your computer, Raspberry Pi, or any other device that supports Docker. This method does not require any coding, and configuration is done through a web interface.
+
+1. Install Docker on your computer. See [Docker's installation instructions](https://docs.docker.com/get-docker/).
+2. Create a new file called `docker-compose.yml` in your project directory. Copy the following into it:
+
+```yaml
+services:
+  frontend:
+    image: docker.io/maxatgroundlight/groundlight-edge-client-frontend:latest
```

Comment:
We should move this to the Groundlight dockerhub account, and use a consistent name for the image - `detector-builder` seems like a good choice.

Reply:
Agreed! I believe that that's what the meeting tomorrow is for.

In the mean time I don't have settings access to the repo, so if you're itching to change the name go for it! Detector-Builder is much much better.